### PR TITLE
7.x 2.x soe 3038

### PR DESCRIPTION
**Pull request by minorwm**
# READY FOR REVIEW

# Summary
This adds the Injector file 50 into SCSS.

# Needed By (Date)
- Sprint end

# Criticality
- Collections Injector file 50 into SCSS

# Steps to Test
- Switch to this branch  -  7.x-2.x-SOE-3038 (https://github.com/SU-SOE/stanford_soe_helper/tree/7.x-2.x-SOE-3038)
- edit the injector: `admin/config/development/css-injector/edit/50` and disable it.
- `drush cc css-js`
- go to: `magazine/article/jeremy-bailenson-taking-grand-tour-latest-virtual-reality` and make sure the changes look good.

# Injector code and screen shots
`.dm-global-header #site-title-first-line.site-title-uppercase,
.dm-global-header #site-title-first-line.site-title-uppercase a {
    font-size: 18px;
}`

screen shot 2018-06-19 at 8 51 34 am

`a {
    font-weight: 400;
}`

screen shot 2018-06-19 at 9 08 46 am

`.view-stanford-magazine-article-mag-landing-page .mag-article-container .mag-article-content-container .mag-article-title a, 
.view-stanford-magazine-collection-mag-landing-page .mag-article-container .mag-article-content-container .mag-article-title a, 
.view-display-id-article_collection_bottom_block .mag-article-container .mag-article-content-container .mag-article-title a {
  font-weight: 600;
}`

screen shot 2018-06-19 at 9 19 35 am

`.view-stanford-magazine-article-most-recent.article-grouping .mag-topic-card-container .mag-article-title[class*="mag-article-color-"] a, 
.view-stanford-magazine-articles.article-grouping .mag-topic-card-container .mag-article-title[class*="mag-article-color-"] a, 
.view-stanford-magazine-topics.article-grouping .mag-topic-card-container .mag-article-title[class*="mag-article-color-"] a,
.view-stanford-magazine-issues .mag-article-container .mag-article-content-container .mag-article-title[class*="mag-article-color-"] a,
.view-stanford-magazine-issues .mag-feat-article-container .mag-article-content-container .mag-article-title[class*="mag-article-color-"] a,
.view-stanford-magazine-article-most-recent.most-recent-article .most-recent-article-container .mag-topic-card-container .mag-article-title[class*="mag-article-color-"] a, 
.view-stanford-magazine-articles.most-recent-article .most-recent-article-container .mag-topic-card-container .mag-article-title[class*="mag-article-color-"] a, 
.view-stanford-magazine-topics.most-recent-article .most-recent-article-container .mag-topic-card-container .mag-article-title[class*="mag-article-color-"] a{
  font-weight: 600;
}`
screen shot 2018-06-18 at 5 35 03 pm

`#digital-magazine-menu .region-digital-magazine-menu .block-menu-block ul li a {
  font-weight: 600;
}`
screen shot 2018-06-18 at 5 38 54 pm

# Affects 
- SoE (School of Engineering website)

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/SOE-3038

## Folks to notify
@cjwest and @boznik 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: scss/components/_soe_brand_bar.scss [hunk 1]
```diff
@@ -4,6 +4,38 @@
...
+        }
+        @include breakpoint-max(xxx-small) {
+          font-size: 16px;
+          margin-bottom: -4px;
+        }
+      }
+    }
+  }
+}
+
```
Comment (cjwest) [inline scss/components/_soe_brand_bar.scss]: @minorwm,

I think this should be in _soe_dm_brand_bar.scss.

File: scss/mixins/_soe_breakpoints.scss [hunk 2]
```diff
@@ -50,4 +53,7 @@
   @else if $point == xx-small {
     @media (min-width: 400px)  { @content; }
   }
+  @else if $point == xx-small {
```
Comment (cjwest) [inline scss/mixins/_soe_breakpoints.scss]: @minorwm,

Did you mean to have xxx-small here?

File: scss/components/_soe_dm_brand_bar.scss [hunk 3]
```diff
@@ -41,3 +41,35 @@
     }
   }
 }
+
+a {
+  font-weight: 400;
+}
+
```
Comment (cjwest) [inline scss/components/_soe_dm_brand_bar.scss]: This has no other selectors so it sets "a" globally. I don't think this belongs here.

File: scss/components/_soe_dm_brand_bar.scss [hunk 4]
```diff
@@ -41,3 +41,35 @@
...
+          margin-bottom: -3px;
+        }
+        @include breakpoint-max(xxx-small) {
+          font-size: 16px;
+          margin-bottom: -4px;
+        }
+      }
+    }
+  }
+}
```
Comment (cjwest) [inline scss/components/_soe_dm_brand_bar.scss]: @minorwm 
This give us _two_ places to maintain the brand bar. What about just update the code starting at line 7?

**Review by cjwest — CHANGES_REQUESTED** [review]
Hi @minorwm,

Two more comments for you.

File: scss/components/_soe_dm_brand_bar.scss (hunk 3 above)
Comment (minorwm) [reply scss/components/_soe_dm_brand_bar.scss]: it was in the injector: https://engineering.stanford.edu/admin/config/development/css-injector/edit/50

I just added what was in there.

**Review by cjwest — APPROVED** [review]

**Comment by cjwest** [comment]
Lookin' good Mr. @minorwm!

Nice pics!

**Pull request merged**